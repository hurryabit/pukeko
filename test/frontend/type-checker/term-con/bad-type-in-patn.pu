data T a b = | C a b
f : T a b -> a
f t = match t with
  | C x y -> y
