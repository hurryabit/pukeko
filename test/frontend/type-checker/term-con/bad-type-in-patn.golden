test/frontend/type-checker/term-con/bad-type-in-patn.pu:3:1: mismatching types a and b
